tools/props/Versions.props

```diff
@@ -6,7 +6,7 @@
     <!-- Should only be changed in future when a non-backwards compatible driver is released. -->
     <!-- Future Assembly Version values shall be Major.Minor.0.0; e.g. 4.0.0.0 -->
     <AssemblyVersion>5.0.0.0</AssemblyVersion>
-    <NugetPackageVersion Condition="'$(NugetPackageVersion)' == ''">5.0.0-dev</NugetPackageVersion>
+    <NugetPackageVersion Condition="'$(NugetPackageVersion)' == ''">5.0.0-preview1.22062.1</NugetPackageVersion>
```

Comment:
This is the MDS NuGet version, not SNI. Please, revert it.

Reply:
Corrected.